Dalamud/Plugin/Services/IDtrBar.cs

```diff
@@ -24,11 +26,13 @@ public interface IDtrBar
     /// <param name="text">The text the entry shows.</param>
     /// <returns>The entry object used to update, hide and remove the entry.</returns>
     /// <exception cref="ArgumentException">Thrown when an entry with the specified title exists.</exception>
-    public IDtrBarEntry Get(string title, SeString? text = null);
+    IDtrBarEntry Get(string title, SeString? text = null);
 
     /// <summary>
     /// Removes a DTR bar entry from the system.
     /// </summary>
     /// <param name="title">Title of the entry to remove.</param>
-    public void Remove(string title);
+    /// <remarks>Remove operation is not immediate. Attempts to call <see cref="Get"/> immediately after calling this
```

Comment:
How does it fail? How does it need to be handled by the plugin? Can we just revive the entry instead in Get so that the removal is a no-op or is it a multi-step process?

Reply:
That was describing the previous behavior; changed so that remove/get pairs will always work.